modelon/impact/client/entities/workspace.py

```diff
@@ -407,6 +413,20 @@ def size(self) -> float:
     def definition(self) -> WorkspaceDefinition:
```

Comment:
Please add a docstring and indicate that this is a client side object (may be off-sync with the server).

Reply:
Hmm I think we should aim to never let the user worry about not being in sync.. but I am open to discussion :)